### ISSUE TITLE
Better pop-up subtitle 

### ISSUE THREAD

**Issue by Discostu36** [babykarte/babykarte.github.io#46]
Subtitles of pop-ups shouldn't name the filter ("changing tables") but the main tag of the POI ("chemist"). Ideally, we'd have a list where we can easily add "translations":

```
EN: {
"shop=chemist": "Chemist";
"amenity=toilet": "Toilet"; 
}

DE: {
"shop=chemist": "Drogerie";
"amenity=toilet": "WC";
}
```
Filter name could still be a fall back for keys that are not listed.

**Comment by ValorNaram** [comment]
I will do this as third at weekend

**Comment by ValorNaram** [comment]
Now change the transitions. I also have made some jokes so you don't get bored :)

**Comment by Discostu36** [comment]
This doesn't quite work yet. Filter is still shown as sub title (tested with child care and diaper)

**Comment by Discostu36** [comment]
No, still doesn't work. :-)

![grafik](https://user-images.githubusercontent.com/30999326/55586080-6701f280-5728-11e9-90b0-6e5fe5817356.png)

**Comment by ValorNaram** [comment]
Can't reproduce this problem

**Comment by Discostu36** [comment]
http://localhost:8080/#19&54.324401566139166&10.124362438982645

Select diaper and click the POI

**Comment by ValorNaram** [comment]
Oh yes because the POI fulfills the requirement of two filters.

**Comment by Discostu36** [comment]
Maybe just completely separate sub-title from filters?

**Comment by Discostu36** [comment]
Problem persisting (or reappeared?)

![image](https://user-images.githubusercontent.com/30999326/57574497-58e77600-743a-11e9-8eb2-6fd5aeb5fca0.png)

**Comment by ValorNaram** [comment]
Feature request solved. Now set the text you find appropiate.

**Comment by Discostu36** [comment]
![grafik](https://user-images.githubusercontent.com/30999326/58473965-32b81a80-814a-11e9-83b1-e434dbd4b01c.png)

Still showing the filter name instead of main tag.

**Comment by ValorNaram** [comment]
Yes, it occurrs due to a misinterpretation of your specification I had in the past. Reworking of code that makes it happen planned.

**Comment by ValorNaram** [comment]
**Done!**

Changing and adding subtitles is quite easy:

## Adding subtitles to the end
Just add

    ,
   		 "tag=key": "Subtitle"

at the end of the "filtertranslations" list:

    "shop=chemist": "Chemist",

=

    "shop=chemist": "Chemist",
    "tag=key": "Subtitle"

## Adding subtitles to the start
Just add

   		 "tag=key": "Subtitle",

at the start of the "filtertranslations" list:

    "healthcare=doctor": "Pediatrician",

=

    "tag=key": "Subtitle",
    "healthcare=doctor": "Pediatrician",

## Adding subtitles to the middle (anywhere in the "filtertranslations" section)
Just add

   		 "tag=key": "Subtitle",

anywhere in the "filtertranslations" section

like

   "amenity=toilets": "Toilets",
   "tag=key": "Subtitle",

or

    "tag=key": "Subtitle",
    "leisure=playground": "Playground",

or

    "shop=baby_goods": "Baby goods",
     "tag=key": "Subtitle",

**Comment by ValorNaram** [comment]
Refer also to https://www.tutorialspoint.com/json/json_quick_guide.htm for a simple tutorial on JSON

**Comment by Discostu36** [comment]
Still not working 😇
